scripts/commands/checkQiskitBotFiles.ts

```diff
@@ -64,20 +75,14 @@ const ALLOWED_OWNERLESS_FILES = new Set([
 ]);
 
 const ALLOWED_NONEXISTENT_FILES = new Set([
-  "docs/migration-guides/index",
-  "docs/migration-guides/qiskit-runtime",
-  "docs/migration-guides/qiskit-runtime-examples",
-  "docs/migration-guides/qiskit-quantum-instance",
-  "docs/migration-guides/qiskit-algorithms-module",
-  "docs/migration-guides/qiskit-opflow-module",
   // Todo: remove or rename the following old paths
   "docs/build/qubit-order",
   "docs/build/operators_overview",
   "docs/run/reserve-system-time",
```

Comment:
The To-Do can also be removed from `ALLOWED_OWNERLESS_FILES`, `ALLOWED_NONEXISTENT_FILES`, and `GLOBS`. I left them out of https://github.com/Qiskit/documentation/pull/1653 to avoid any merge conflict

Reply:
Will do in https://github.com/Qiskit/documentation/pull/1637